Cargo.toml

```diff
@@ -23,9 +23,8 @@ required-features = ["async-tokio"]
 bitflags = "1.0"
 libc = "0.2"
 nix = "0.14"
-tokio = { version = "0.2", features = ["io-driver", "rt-threaded", "macros"], optional = true }
+tokio = { version = "1", features = ["io-std", "rt-multi-thread", "macros", "net"], optional = true }
```

Comment:
not that this isn't equivalent to the previous version but, i don't believe we should be setting `rt-multi-threaded` here as this library shouldn't impose a runtime configuration.

Reply:
Yes, the runtime flag should probably be removed, just followed the current setting. To get the examples to build one can set tokio with runtime in dev-dependencies. Would that be appropriate?

Reply:
Removed `rt-multi-thread` and `macros` since both are only needed  in the example. And added another dev-dependency on tokio with these flags.

Reply:
great, thanks!